styling/components/button.css

```diff
@@ -0,0 +1,39 @@
+.simple-btns{
+    gap:1rem;
+}
+
+.btn{
+    border-radius: 8px;
+    padding:1rem;
+    font-size: 16px;
+    font-weight: bold;
```

Comment:
Can we add `cursor: pointer;` here on the buttons

Reply:
done ..check now